src/models/create_token_transaction.ts

```diff
@@ -127,19 +127,23 @@ class CreateTokenTransaction extends Transaction {
     array.push(symbolBytes);
   }
 
-  getTokenInfoFromBytes(buf: Buffer): Buffer {
+  getTokenInfoFromBytes(_buf: Buffer): Buffer {
     let tokenInfoVersion;
     let lenName;
     let lenSymbol;
     let bufName;
     let bufSymbol;
+    // Copies buffer locally, not to change the original parameter
+    let buf = Buffer.from(_buf);
 
+    // eslint-disable-next-line prefer-const -- To split this declaration would be confusing
```

Comment:
How confusing would it be?
And maybe instead of disable-next-line, we should disable prefer-const on the function or file so we dont have so many disable comments

Reply:
We would need to, for each call of `unpackToInt()`, devise a way to store the first element with a `const` and the second element in a `let`.

This could be done by either:
- Calling the method once and storing its full response in a variable and then using two destructuring assignments from it, or;
- Calling the method twice and using a different destructuring assignment for each call

I think this is not a reasonable way to write, so I prefer disabling this rule when calling `unpackToInt` for now.

As for the disabling command, it was refactored to be less repetitive on https://github.com/HathorNetwork/hathor-wallet-lib/pull/679/commits/2c120faba11bf808b123074a66bbf743748d3d75